packages/core/src/utils/spanUtils.ts

```diff
@@ -26,6 +28,11 @@ import { _getSpanForScope } from './spanOnScope';
 export const TRACE_FLAG_NONE = 0x0;
 export const TRACE_FLAG_SAMPLED = 0x1;
 
+// todo(v9): Remove this once we've stopped dropping spans via `beforeSendSpan`
+const SPAN_DROP_WARNING =
+  'Dropping spans via `beforeSendSpan` will be removed in SDK v9.0.0. The callback will only support modifying span attributes.';
```

Comment:
Nit: there is very little gain in putting this into a const if we only use it in one place, instead I would just inline it.